internal/plugin/slo/contrib/rule_intervals_v1/README.md

```diff
@@ -0,0 +1,46 @@
+# sloth.dev/contrib/rule_intervals/v1
+
+This plugin sets Prom rule evaluation intervals to the Prometheus generated rules. The intervals can be different depending on the type of rules, SLI, metadata and alerts. A default interval can be set for all rules.
+
+## Config
+
+| Field             | Type                     | Required | Default | Description                                                       |
+|-------------------|--------------------------|----------|---------|-------------------------------------------------------------------|
+| `interval.default` | Prom time duration string  | Yes      | —       | Fallback interval to use if no other interval is set.             |
+| `interval.sliError`| Prom time duration string  | No       | —       | Evaluation rule interval for generated SLI error rules.           |
+| `interval.metadata`| Prom time duration string  | No       | —       | Evaluation rule interval for generated metadata rules.            |
+| `interval.alert`   | Prom time duration string  | No       | —       | Evaluation rule interval for generated alert rules.               |
+
+## Env var
+
+None
+
+## Order requirement
+
+This plugin must be placed **after** all rule generation.
+
+## Usage examples
+
+### Specific settings in an SLO group
+
+```yaml
+slo_plugins:
+  chain:
+    - id: sloth.dev/contrib/validate_victoria_metrics/v1 # Custom validation.
```

Comment:
This should use `sloth.dev/contrib/rule_intervals/v1` ; the comment should also probably be updated :)

Reply:
Thanks @multani! No PRs without typos 😆